pkg/apply/desiredset_compare.go

```diff
@@ -45,6 +44,10 @@ var (
 	}
 )
 
+// prepareObjectForCreate returns a copy of the object with the "applied"
+// annotation set. The 'applied' annotation contains a serialized, pruned
+// representation of obj, e.g. Strings are shortened to 64 characters and bytes
+// are skipped.
```

Comment:
edit: update for quotes to be consistent with your other comments.
```suggestion
// prepareObjectForCreate returns a copy of the object with the "applied"
// annotation set. The "applied" annotation contains a serialized and pruned
// representation of obj, e.g. Strings are shortened to 64 characters and bytes
// are skipped.
```